plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/CompareView.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.ext.git.client.compare;
+
+import com.google.inject.ImplementedBy;
+
+import org.eclipse.che.ide.api.mvp.View;
+
+/**
+ * The view of {@link ComparePresenter}.
+ *
+ * @author Igor Vinokur
+ */
+@ImplementedBy(CompareViewImpl.class)
+interface CompareView extends View<CompareView.ActionDelegate> {
+    /**
+     * Set a title of the window.
+     *
+     * @param title the name of the compare window
+     */
+    void setTitle(String title);
+
+    /** Hide compare panel. */
+    void hide();
+
+    /**
+     * Show compare panel with specified contents.
+     * 
+     * @param oldContent content from specified working tree
```

Comment:
Can you format this javadoc in accordance to https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#javadoc